crates/services/p2p/src/ports.rs

```diff
@@ -21,6 +21,12 @@ pub trait P2pDb: Send + Sync {
         height: &BlockHeight,
     ) -> StorageResult<Option<SealedBlockHeader>>;
 
+    fn get_sealed_headers_range_inclusive(
+        &self,
+        start_height: &BlockHeight,
+        end_height: &BlockHeight,
+    ) -> StorageResult<Vec<SealedBlockHeader>>;
+
```

Comment:
You don't need to have a `_range_inclusive` suffix. Instead, you can work directly with the `Range<BlockHeight>` or `Range<u32>` type